### ISSUE TITLE
Template support for annotation files

### ISSUE THREAD

**Issue by Taitava** [schlundd/obsidian-ocr-plugin#3]
Hi,

would it be possible to implement template support for annotation files, so that every time an annotation file is created, it could have some predefined text in it?

A few different ideas how to implament this (I don't know which ones of these are technically possible or easiest to do):
 - Use Obsidian's core plugin `Templates` to render the template?
 - Use a third party plugin called [Templater](https://github.com/SilentVoid13/Templater)? Or some other third party plugin. Maybe problematic because introduces a new dependency.
 - Make your own, very simple template system?

Variables that the template could support:
- `{{image_path}}`: This would be the inner content of the `![[]]` element that the current annotation files have on the very top of them.
- `{{annotation_text}}`: The text that came from the OCR service.
- \* `{{title}}`: Image file's name, maybe without the file extension (= "Shopping list.jpg" would become just "Shopping list") 
- \* `{{date}}`: Date when the annotation was created.
- \* `{{time}}`: Time when the annotation was created.
*) These variables use the same names as Obsidian's core plugin `Templates`, so they should be quite familiar to many users.

An example template file:
```markdown
# Annotation for {{title}}
{{annotation_text}}

# Original image
![[{{image_path}}]]

(Annotation was created on {{date}} {{time}).
````

Thanks! :)

**Comment by schlundd** [comment]
Thanks for the detailed explanation!

I think this will work, but maybe with a twist. I'll get back to you.

**Comment by schlundd** [comment]
Basically the same as with your other request... Finally possible in the new plugin. The new command can be configured with a template in the settings.

Your example would look like this:
```
# Annotation for {{ imageFileBaseName }}
{{ text }}

# Original image
![[{{ imageFilePath }}]]

(Annotation was created on {{ "now" | date: "%Y-%m-%d %H:%M" }}).
```
Let me know, if you need any hekp setting it up.

**Comment by Taitava** [comment]
What do you mean by "the new plugin"? 🙂 Is it something you are developing at the moment, or is it already available somewhere?

Anyway, nice to know that this has been processed onwards. Thanks! 🙂

**Comment by schlundd** [comment]
Finally solved with the recent release of 2.0.0
